test/unit/lib/gpHelper.js

```diff
@@ -98,4 +98,42 @@ describe('gp counts helper', () => {
       expect(message[0]).to.equal('1 GP');
     });
   });
+
+  describe('getPersonSingular', () => {
+    it('should return \'are\' for more than one male and female gp', () => {
+      const gpCounts = {
+        male: 3,
+        female: 2
+      };
+      const personSingular = gpHelper.getPersonSingular(gpCounts);
+      expect(personSingular).to.equal('are');
+    });
+
+    it('should return \'is\' for only one male gp', () => {
+      const gpCounts = {
+        male: 1,
+        female: 0
+      };
+      const personSingular = gpHelper.getPersonSingular(gpCounts);
+      expect(personSingular).to.equal('is');
+    });
+
+    it('should return \'is\' for only one female gp', () => {
+      const gpCounts = {
+        male: 0,
+        female: 1
+      };
+      const personSingular = gpHelper.getPersonSingular(gpCounts);
+      expect(personSingular).to.equal('is');
+    });
+
+    it('should return \'is\' for one female gp and one male GP', () => {
```

Comment:
Can this test be changed to `should return 'is\ for one female gp and any number of male GPs` to high light the fact that if there is only a single female GP the return will always be `is`. I'd probably have a look within the test that incremented the number of male GPs from 0 to 2 and run the same `expect(...)` as is already here. The test above would then become obsolete.
Another option would be to add an additional test and reword the 3 (this one, the new one and the one above) to be clearer. However, my preference would be the first suggestion.